### PR TITLE
rustup:add page

### PR DESCRIPTION
**Pull request by zethra**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

**Comment by sbrl** [comment]
Hey - thanks for the new page, @zethra! Looks like you're having some issues with the linter. We've got the style we prefer documented over [here](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#markdown-format), and a few good examples are the pages for `tar`, `eyeD3`, and `ln`. If you need any assistance, just ask :smiley_cat:

File: pages/common/rustup.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`rustup update`
+
+- List installed toolchains:
+
+`rustup show`
+
+- Run cargo build with a certain toolchain:
+
+`rustup run {{toolchain name}} cargo build`
```
Comment (sbrl) [inline pages/common/rustup.md]: We usually prefer `snake_case`. Could you change this to be `toolchain_name` plese?

Comment (zethra) [reply pages/common/rustup.md]: Sure

File: pages/common/rustup.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# rustup
+
+> Rust toolchain installer.
+> Install, manage, and update Rust toolchains.
+
+- Install the nightly toolchain for your system:
+
+`rustup install nightly`
```
Comment (agnivade) [inline pages/common/rustup.md]: What does `rustup install` install by default ? We should probably start with the simplest form of the command and then build on to more complex versions.

File: pages/common/rustup.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Switch the default toolchain to nightly so that the cargo and rustc commands will use it:
```
Comment (agnivade) [inline pages/common/rustup.md]: cargo and rustc within backticks.

File: pages/common/rustup.md (hunk 2 above)
Comment (zethra) [reply pages/common/rustup.md]: `rustup install` prints and error by default.  You have to specify a toolchain.

File: pages/common/rustup.md (hunk 3 above)
Comment (zethra) [reply pages/common/rustup.md]: Will do.

File: pages/common/rustup.md (hunk 2 above)
Comment (agnivade) [reply pages/common/rustup.md]: Got it. Thanks for the clarification.

**Comment by sbrl** [comment]
Thanks, @zethra!

**Pull request merged**